spec/views/application_records/show_spec.rb

```diff
@@ -0,0 +1,160 @@
+              with_tag 'a', with: { href: action_path }
+            end
+          end
+          it 'contains a form to reschedule an interview' do
+            render
+            action_path = reschedule_interview_path @interview
+            expect(rendered).to have_form action_path, :post do
+              with_tag 'input#scheduled.datetimepicker'
+              with_text_field :location, @interview.location
+            end
+          end
+          it 'contains a form to mark interview as complete' do
+            render
+            action_path = complete_interview_path @interview
+            expect(rendered).to have_form action_path, :post
+          end
+        end
+        context 'interview is complete' do
+          before :each do
+            @interview.update completed: true
+            assign :interview, @interview
+          end
+          it 'shows when and where the interview occurred' do
+            render
+            expect(rendered).to include @interview.information
+          end
+        end
+      end
+    end
+  end
+  context 'current user is student' do
+    before :each do
+      when_current_user_is :student, view: true
+    end
+    context 'application not reviewed' do
+      before :each do
+        @record.update reviewed: false
+        assign :record, @record
+      end
+      it 'explains that the application has not been reviewed' do
+        render
+        expect(rendered).to include 'has not yet been reviewed'
+      end
+    end
+    context 'application reviewed' do
+      before :each do
+        @record.update reviewed: true
+        assign :record, @record
+      end
+      context 'interview present' do
+        before :each do
+          @interview = create :interview
+        end
+        context 'interview is not complete' do
+          before :each do
+            @interview.update completed: false
+            assign :interview, @interview
+          end
+          it 'contains text saying that the interview is scheduled' do
+            render
+            expect(rendered).to include 'interview is scheduled for'
+          end
+          it 'contains the time, date and location of the interview' do
+            render
+            expect(rendered).to include @interview.information
+          end
+        end
+        context 'interview is complete' do
+          before :each do
+            @interview.update completed: true
+            assign :interview, @interview
+          end
+          it 'contains text saying that the interview has occurred' do
+            render
+            expect(rendered).to include 'interview occurred on'
+          end
+          it 'contains the time, date and location of the interview' do
+            render
+            expect(rendered).to include @interview.information
+          end
+        end
+      end
+      context 'interview not present' do
+        before :each do
+          @record.interview.delete if @record.interview.present?
```

Comment:
I'd like to clean up this line a little. We shouldn't need conditional logic in tests, we should be able to say confidently what the state of our data is.